### PR TITLE
OCPBUGS-14294: Remove erroneous text

### PR DESCRIPTION
**Pull request by jab-rh**
- https://issues.redhat.com/browse/OCPBUGS-14294

Version(s): 4.12+

Issue:

Link to docs preview:

[Cluster Network Operator configuration](https://61106--docspreview.netlify.app/openshift-enterprise/latest/networking/cluster-network-operator.html#nw-operator-cr_cluster-network-operator)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@zhaozhanqi, can you confirm? Thanks!

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -292,8 +292,6 @@ endif::ibm-cloud[]
 |
 If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be larger than the maximum number of nodes that can be added to the cluster.
 
-For example, if the `clusterNetwork.cidr` is `10.128.0.0/14` and the `clusterNetwork.hostPrefix` is `/23`, then the maximum number of nodes is `2^(23-14)=128`. An IP address is also required for the gateway, network, and broadcast addresses. Therefore the internal IP address range must be at least a `/24`.
```
Comment (zhaozhanqi) [inline modules/nw-operator-cr.adoc:295]: I think we just need to update `2^(23-14)=128` to `2^(23-14)=512`,  why removing all?

Comment (zhaozhanqi) [reply modules/nw-operator-cr.adoc:295]: yes, it makes sense to me

**Comment by zhaozhanqi** [comment]
/hold 
waiting the update according to the comment

**Comment by jab-rh** [comment]
@zhaozhanqi, I've added the update.

File: modules/nw-operator-cr.adoc [hunk 2]
```diff
@@ -288,9 +288,7 @@ endif::ibm-cloud[]
...
-
-For example, if the `clusterNetwork.cidr` is `10.128.0.0/14` and the `clusterNetwork.hostPrefix` is `/23`, then the maximum number of nodes is `2^(23-14)=512`. An IP address is also required for the gateway, network, and broadcast addresses. Therefore the internal IP address range must be at least a `/24`.
+If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be larger than the maximum number of nodes that can be added to the cluster. For example, if the clusterNetwork.cidr is `10.128.0.0/14` and the clusterNetwork.hostPrefix is `/23`, then the maximum number of nodes is `2^(23-14)=512`.
```
Comment (jeana-redhat) [inline modules/nw-operator-cr.adoc:291]:
```suggestion
If your existing network infrastructure overlaps with the `100.64.0.0/16` IPv4 subnet, you can specify a different IP address range for internal use by OVN-Kubernetes. You must ensure that the IP address range does not overlap with any other subnet used by your {product-title} installation. The IP address range must be larger than the maximum number of nodes that can be added to the cluster. For example, if the `clusterNetwork.cidr` value is `10.128.0.0/14` and the `clusterNetwork.hostPrefix` value is `/23`, then the maximum number of nodes is `2^(23-14)=512`.
```

**Review by jeana-redhat — COMMENTED** [review]
Li'l nit, otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by jab-rh** [comment]
@zhaozhanqi, does this look okay? Thanks!

**Pull request merged**